frameworks/Python/API-Hour/benchmark_config

```diff
@@ -20,8 +20,62 @@
       "webserver": "None",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "API-Hour",
-      "notes": "Python 3.4 + AsyncIO"
+      "display_name": "API-Hour+aiohttp.web+pgsql",
+      "notes": "Python 3 + AsyncIO + aiohttp.web + PostgreSQL"
+    }
```

Comment:
Thank you, I've fixed that in the pull request.
I've checked the syntax with jsonlint.
If for round 11, the file extension should be .json, it will be better to detect issues in IDE directly.